ansible/group_vars/all/openstack

```diff
@@ -48,6 +48,28 @@ openstack_image_elements: []
 
 # List of diskimage-builder element git repos to pull
 openstack_image_git_elements: []
+###############################################################################
+# Configuration variables for a CloudKitty ratings service deployment.
+
+# File path or URL containing Python upper constraints.
+#
+ratings_upper_constraints_file: []
+
+# Environment variables for the OpenStack CLI. By default this is left empty.
+#
+ratings_environment: []
```

Comment:
I don't think you need to define these here, the defaults from the role should be sufficient.

Reply:
These will error out as the `openstack-ratings.yml` playbook will request these variables before the role is called upon and being able to provide values. Additionally, giving the user the option to include them means that they should also provide a default.

Reply:
Responded in the other comment thread:
> I was thinking along the lines of matching the conventions the other roles use. I can't immediately see why we'd want to override these values often, but I guess that it doesn't hurt to expose them.